Source/FakeItEasy.Tests/Core/DefaultSutInitializerTests.cs

```diff
@@ -57,17 +57,27 @@ public void Should_use_same_instance_when_multiple_arguments_are_of_same_type()
         }
 
         [Test]
-        public void Should_pass_empty_fake_options_to_fake_manager()
+        public void Should_pass_impotent_options_builder_to_fake_manager_for_each_constructor_argument()

```

Comment:
Well, the use of the term "impotent" surprised me, but I understand what you meant by it. Perhaps "empty" would have a more obvious meaning, but I don't really care either way.